storage/innobase/handler/handler0alter.cc

```diff
@@ -5374,7 +5378,8 @@ prepare_inplace_alter_table_dict(
 				ut_d(const dict_index_t* index
 				     = user_table->indexes.start);
 				DBUG_ASSERT(col->mtype == old_col->mtype);
-				DBUG_ASSERT(col->prtype == old_col->prtype);
+				DBUG_ASSERT(col->prtype == old_col->prtype ||
```

Comment:
Binary operator should be at the beginning of the next line in InnoDB coding style. See next assert, f.ex.

Reply:
Fixed.